docs/ref/databases.txt

```diff
@@ -1082,6 +1082,46 @@ Example of a full DSN string::
         "(CONNECT_DATA=(SERVICE_NAME=orclpdb1)))"
     )
 
+.. _oracle-pool:
+
+Connection Pool
+---------------
+
+.. versionadded:: 5.1
+
+To use a connection pool with `oracledb`_, you can set the ``"pool"`` in the
+:setting:`OPTIONS` part of your database configuration in :setting:`DATABASES`::
+
+    DATABASES = {
+        "default": {
+            "ENGINE": "django.db.backends.oracle",
+            # ...
+            "OPTIONS": {
+                "pool": True,
+            },
+        },
+    }
+
+.. _`oracledb`: https://oracle.github.io/python-oracledb/
+
+You can pass all the parameters accepted by `create_pool()`_ through :setting:`OPTIONS`
+part of your database configuration :setting:`DATABASES`::
+
+    DATABASES = {
+        "default": {
+            "ENGINE": "django.db.backends.oracle",
+            # ...
+            "OPTIONS": {
+                "pool": True,
+                "min": 1,
+                "max": 10,
```

Comment:
I'd recommend to use the same pattern as in #17914.

Reply:
This is not fixed, it should be:
```
"pool": {
    "min": 1,
    "max": 10,
}
```
Also, I cannot find any place where pool's parameters are used.

Reply:
You are suggesting "pool" can have 2 values .
1 - True/False (boolean)
2 - { "min":1, "max":2 } (Dictionary)